external/vpx-jpattackfrommars/README.md

```diff
@@ -0,0 +1,32 @@
+# Attack From Mars (Bally 1995) vpx Graphics Remaster
+
+![Table Preview](../../images/vpx-jpattackfrommars-preview.png)
+
+---
+
+## Files
+| File Type |                                                 Link                                                  |   Version    |                                                          Author                                                         |
```

Comment:
Remove extra spaces in the header. There should be one space before and after each header.

Reply:
Addressed table formatter issue.